### PR TITLE
Transfer logger

### PR DESCRIPTION
**Pull request by sigu**

### CODE REVIEW

**Review by sparrell — CHANGES_REQUESTED** [review]
It fails tests due to missing clientid. can you set it up like the others which I don't think have this issue (ie test env variables)

File: .github/workflows/elixir.yml [hunk 1]
```diff
@@ -47,5 +47,11 @@ jobs:
       run: mix deps.get
     - name: Run tests
       run: mix test
+      env:
+        CLIENT_ID: :sfractal2020
```
Comment (sparrell) [inline .github/workflows/elixir.yml:51]: is this a default or a hardcoded value? 
Eitherway, can we use :sbompoc for this repo (ie don't use same for each repo)?

File: Dockerfile [hunk 2]
```diff
@@ -36,6 +36,20 @@ RUN mix phx.digest && mix release
 
 FROM debian:buster-slim AS app
 
+ARG CLIENT_ID=:sfractal2020
```
Comment (sparrell) [inline Dockerfile:39]: can we use :sbompoc for this repo (ie don't use same for each repo)?

**Review by sparrell — CHANGES_REQUESTED** [review]

File: .github/workflows/elixir.yml (hunk 1 above)
Comment (sigu) [reply .github/workflows/elixir.yml:51]: This is for use in test environment only on github

File: Dockerfile (hunk 2 above)
Comment (sigu) [reply Dockerfile:39]: Yeah, sure
```suggestion
ARG CLIENT_ID=:sbompoc
```

File: Makefile [hunk 3]
```diff
@@ -5,6 +5,11 @@ APP_NAME ?= `grep 'app:' mix.exs | sed -e 's/\[//g' -e 's/ //g' -e 's/app://' -e
 APP_VERSION := $(shell grep 'version:' mix.exs | cut -d '"' -f2)
 DOCKER_IMAGE_TAG ?= $(APP_VERSION)
 GIT_REVISION ?= `git rev-parse HEAD`
+CLIENT_ID=:sbom_poc
```
Comment (sigu) [inline Makefile:8]:
```suggestion
CLIENT_ID=:sbompoc
```

**Review by sparrell — APPROVED** [review]
there is now a conflict. I think I caused it by approving the other sbom pr and merging it in. I'll let you resolve this conflich and do the merge so I don't screw it up further.

**Pull request merged**